WcaOnRails/app/views/persons/_medal_collection.html.erb

```diff
@@ -10,9 +10,21 @@
     </thead>
     <tbody>
       <tr>
-        <td><%= @medals[:gold] %></td>
```

Comment:
FYI you can use rails helpers here to make your life a little easier:
```html
<td><%= link_to @medals[:gold], "#", class: "highlight-medal", data: { place: "gold" } %></td>
```

I'm not sure if it's worth to change it, your choice :)